src/test/java/ch/jalu/configme/migration/PlainMigrationServiceTest.java

```diff
@@ -111,7 +112,7 @@ void shouldMoveProperty() {
         ConfigurationData configurationData = mock(ConfigurationData.class);
 
         // when
-        boolean result = PlainMigrationService.moveProperty(
+        boolean result = MigrationUtils.moveProperty(
```

Comment:
Please move to a new class `MigrationUtilsTest`. I can do it after the PR otherwise, if it helps